cypress/integration/admin_portal/pages/enterprise_code_management.js

```diff
@@ -56,10 +60,25 @@ class CodeManagementPage {
     return cy.get('.table tbody tr:nth-of-type(1) td:nth-of-type(4)')
   }
 
+  getNumberOfRows() {
+    return cy.get('.table tbody tr')
+  }
+
+  selectCodesForBulkAssignment(rowNumber){
+    return cy.get(`.table tbody tr:nth-of-type(${rowNumber}) .form-check label`).click()
+  }
+
+  selectCodesForBulkRevoke(){
+    return cy.get('label[id="label-select-all-codes"]')
+  }
+
   getModalWindow() {
     return cy.get('.modal-content')
   }
 
+  getCouponNameFromModalWindow(){
+    return cy.get('.modal-title>span')
```

Comment:
if you want to return text just do so
return cy.get('.modal-title>span').invoke('text')

Reply:
removing this method, as it is no longer in use.